build/gen_stub.php

```diff
@@ -67,10 +67,8 @@ function processStubFile(string $stubFile) {
 }
 
 class SimpleType {
-    /** @var string */
-    public $name;
-    /** @var bool */
-    public $isBuiltin;
+    public string $name;
```

Comment:
I added typed properties because make the code more clear, but I can remove them if compatibility with older PHP versions is important.

Reply:
This script is intended to be PHP 7.1 compatible right now.